docs/getting_started/example_configurations.md

```diff
@@ -6,14 +6,20 @@ Example configurations
 
 Following benchmark [configuration]({{page.path_to_root}}benchmark_configuration/general.html) files can be found in `/conf` folder of [built](./building_binaries.html) project.
 
+**NOTE:** When working with external application builds always check you have appropriate version, in case of wierd errors check again.
+
 **[benchmark-aggregation-query.xml](https://github.com/radargun/radargun/blob/master/extensions/query/src/main/resources/benchmark-aggregation-query.xml)** - Shows correct usage of `query` extension on Infinispan 9.0
 
 **[benchmark-analysis.xml](https://github.com/radargun/radargun/blob/master/extensions/cache/src/main/resources/benchmark-analysis.xml)** - Shows how to analyze results of a stage and then re-use the results in another stage/store them to results via `analyze-test` and `add-result` stages
 
 **[benchmark-coherence-hazelcast.xml](https://github.com/radargun/radargun/blob/master/extensions/cache/src/main/resources/benchmark-coherence-hazelcast.xml)** - Shows how to run comparative analysis of three products (Coherence, Hazelcast and Infinispan)
 
+This benchmark requires [Coherence plugin]({{page.path_to_root}}other_docs/coherence_plugin.html) which is not built by default due to licensing constraints.
+
 **[benchmark-combined-report.xml](https://github.com/radargun/radargun/blob/master/extensions/cache/src/main/resources/benchmark-combined-report.xml)** - Shows how to property handle combining different tests
 
+This benchmark is using colliding-keys [key selector]({{page.path_to_root}}benchmark_configuration/key_selectors.html), consequently it produces a **lot** of errors in logs due to key clashes.
```

Comment:
"Could not obtain lock before timeout" I think. I assumed that is part of the game.